components/suggest/src/db.rs

```diff
@@ -0,0 +1,262 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+use std::{
+    path::Path,
+    sync::{Arc, Mutex},
+};
+
+use interrupt_support::SqlInterruptHandle;
+use rusqlite::{
+    named_params,
+    types::{FromSql, ToSql},
+    Connection, OpenFlags,
+};
+use sql_support::{open_database::open_database_with_flags, ConnExt};
+
+use crate::{
+    keyword::full_keyword, schema::SuggestConnectionInitializer, RemoteRecordId, RemoteSuggestion,
+    Result, Suggestion,
+};
+
+/// The metadata key whose value is the timestamp of the last ingested Remote
+/// Settings record.
+pub const LAST_FETCH_META_KEY: &str = "last_fetch";
+
+/// A list of IAB categories that contain non-sponsored suggestions.
```

Comment:
I wonder if `IAB` is worth expanding, and/or adding a link to something which might explain this sorcery?

Reply:
Definitely! I copied this constant from Desktop, but the comment is unhelpful in explaining what an "IAB category" is (Internet Advertising Bureau taxonomy for web pages), or where it comes from (Remote Settings, where it's a property of each suggestion).